crypto/fipsmodule/cipher/e_aes.c

```diff
@@ -237,9 +244,12 @@ int GFp_aes_gcm_init(uint8_t *ctx_buf, size_t ctx_buf_len, const uint8_t *key,
     return 0;
   }
 
-  // XXX: Ignores return value. TODO: These functions should return |void|
-  // anyway.
-  GFp_AES_set_encrypt_key(key, (unsigned)key_len * 8, &ks);
+  unsigned bits = key_len * 8;
+  if (bits != 128 && bits != 256) {
```

Comment:
Instead, let's compare `key_len != 16 && key_len != 32` so we don't have to worry about any overflow issues with the multiplication.